vision/getting-started/image_segmentation.ipynb

```diff
@@ -0,0 +1,1003 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "vbNgv4q1T2Mi"
+      },
+      "outputs": [],
+      "source": [
+        "import sys\n",
+        "\n",
+        "# Additional authentication is required for Google Colab\n",
+        "if \"google.colab\" in sys.modules:\n",
+        "    # Authenticate user to Google Cloud\n",
+        "    from google.colab import auth\n",
+        "\n",
+        "    auth.authenticate_user()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ybBXSukZkgjg"
+      },
+      "source": [
+        "### Set Google Cloud project information and initialize Vertex AI SDK\n",
+        "\n",
+        "To get started using Vertex AI, you must have an existing Google Cloud project and enable the Vertex AI API.\n",
+        "\n",
+        "Learn more about setting up a project and a development environment."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "q7YvbXXdtzDT"
+      },
+      "outputs": [],
+      "source": [
+        "from google.cloud import aiplatform\n",
+        "from google.cloud.aiplatform.gapic import PredictResponse\n",
+        "\n",
+        "PROJECT_ID = \"<your-project-id-here>\"  # @param {type:\"string\"}\n",
+        "LOCATION = \"us-central1\"  # @param [\"asia-northeast1\", \"asia-northeast3\", \"asia-southeast1\", \"europe-west1\", \"europe-west2\", \"europe-west3\", \"europe-west4\", \"europe-west9\", \"northamerica-northeast1\", \"us-central1\", \"us-east4\", \"us-west1\", \"us-west4\"]\n",
+        "\n",
+        "aiplatform.init(project=PROJECT_ID, location=LOCATION)\n",
+        "\n",
+        "api_regional_endpoint = f\"{LOCATION}-aiplatform.googleapis.com\"\n",
+        "client_options = {\"api_endpoint\": api_regional_endpoint}\n",
+        "client = aiplatform.gapic.PredictionServiceClient(client_options=client_options)\n",
+        "\n",
+        "model_endpoint = f\"projects/{PROJECT_ID}/locations/{LOCATION}/publishers/google/models/image-segmentation-001\"\n",
+        "print(f\"Prediction client initiated on project {PROJECT_ID} in {LOCATION}.\")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Ju_PctW22NUl"
+      },
+      "outputs": [],
+      "source": [
+        "# @title Import libraries\n",
+        "# @markdown Run this cell before proceeding to import libraries and define utility functions.\n",
+        "import io\n",
+        "import random\n",
+        "import timeit\n",
+        "\n",
+        "from IPython.display import display\n",
+        "from PIL import Image, ImageDraw, ImageFont\n",
+        "from google.colab import files\n",
+        "from google.protobuf.struct_pb2 import Value\n",
```

Comment:
Don't use a raw Protobuf class for these values. Developers outside of Google generally aren't familiar with these.

Use one of the classes from the client library/Vertex AI Python SDK if possible.

Reply:
Removed